index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/085384e239.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <!----------navbar setion------------------->
+    <nav class="nav-container">
+        <div class="sidebar-right" id="sidebarright">
+            <div class="cross"><a href=""><i class="fa-sharp fa-solid fa-circle-xmark"></i></a></div>
+            <div><a href="">Portfolio</a></div>
+            <div><a href="">About</a></div>
+            <div><a href="">Contact</a></div>
+
+        </div>
+        <h1 class="item-logo">Hyder</h1>
+        <ul class="items-menu">
+            <a href="#">
+                <li>Portfolio</li>
+            </a>
+            <a href="#">
+                <li>About</li>
+            </a>
+            <a href="#">
+                <li>Contact</li>
+            </a>
+            <a href="#">
+                <li><i class="fa-solid fa-envelope"></i></li>
+            </a>
+
+        </ul>
+        <a href="#">
+            <li><i class="fa-solid fa-bars" onclick="openSideBar()"></i></li>
+        </a>
+
+    </nav>
+    <!--------------Hero section------------------------------------------->
+    <div class="hero-section">
+        <div class="hero-row">
+            <div class="hero-left">
+                <a href="#"><i class="fa-brands fa-github"></i></a>
+                <a href="#"><i class="fa-brands fa-linkedin"></i></a>
+                <a href="#"><i class="fa-brands fa-twitter"></i></a>
+                <a href="#"><i class="fa-solid fa-m"></i></a>
+            </div>
+            <div class="hero-right">
+                <h1>Hi There.<br>I am Hyder</h1>
+                <h5>I'm a web Developer</h5>
+                <p>I can help you build a product , feature or website. Look through some of my work and experience! If
+                    you like what you
+                    see and have a project you need coded,
+                    don’t hesitate to contact me.</p>
+            </div>
+            <div class="hero-right1">
+                <img src="Illustration-header.png">
+            </div>
```

Comment:
- [ ] While you have made great effort here, I will suggest that you use CSS `background-image` to show your nice background.  This `image` has little impact and that's because it is not a major content for users' consumption. It is just good for aesthetics - to show visitors what you've got :wink: . So, let's use our stylesheet to bring this in after removing it.